gateway/src/apicast/policy/local_chain/local_chain.lua

```diff
@@ -45,6 +46,9 @@ local rewrite = _M.rewrite
 
 function _M:rewrite(context, ...)
   build_chain(context)
+
+  context = LinkedList.readwrite(context, find_policy_chain(context):export())
```

Comment:
Shouldn't this  call `self:export()` instead of `find_local_policy_chain` ?

Reply:
I don't see it. `export` is not defined in the local chain class.

Reply:
I see. Would it make sense to add it to:
https://github.com/3scale/apicast/blob/85a14f2b5cecb199b856264df1b2e2de6b4cd9da/gateway/src/apicast/policy.lua#L11-L17

Then we should be able to just call it and it would create proper delegator.

Reply:
I'm not sure. The case of `export` is a bit different from the rest of phases.

In the rest of phases, we just call that phase for each of the policies. `export` needs to build a list instead. Like in https://github.com/3scale/apicast/blob/869aa91a3e435cdb1f30cdc5fac698d3adba83f5/gateway/src/apicast/policy_chain.lua#L114

Reply:
Only the PolicyChain merges those exports. The rest of policies just export something. I think that is exactly the same as the other phases. And if every policy would have an export function we could get rid of the `or`: 
https://github.com/3scale/apicast/blob/869aa91a3e435cdb1f30cdc5fac698d3adba83f5/gateway/src/apicast/policy_chain.lua#L120
Making it easier for LuaJIT.

No need to change this. I just thought it would be even nicer cleanup to treat all phases the same.

Reply:
I'd leave this for a future PR.